### PR TITLE
SAK-41198 Don't hide the target site's top level folder

### PR DESCRIPTION
**Pull request by adrianfish**
... when importing content.

### CODE REVIEW

File: kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java [hunk 1]
```diff
@@ -8461,7 +8462,7 @@ else if (edit instanceof ContentCollectionEdit)
 				 * the collection ID. If <= 3, then it's a top-level item and needs to be hidden.
```
Comment (ern) [inline kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java]: need to adjust this comment to match the new logic

**Comment by adrianfish** [comment]
All the files would need unhiding, yes, as opposed to the import basically hiding your entire top level collection. It's good point, though. I could make it a bit smarter so it hides top level imported resources, any collections coming in the import could just be hidden without hiding their contained files.

**Comment by adrianfish** [comment]
@ern I've updated the code to only hide things if they are at the root of the target site's collection hierarchy, so /group/TARGETSITEID/resource1 and /group/TARGETSITEID/collection1 will get hidden on import. Hiding only the 'top' level folders like this takes care of any contained files.

**Comment by adrianfish** [comment]
Assuming this is cool, I'll stick a test plan on the SAK.

**Comment by adrianfish** [comment]
@ern What do you reckon, good to go?

**Pull request merged**